app/res/values/strings.xml

```diff
@@ -364,28 +364,20 @@
     <string name="incorrect_time_dialog_cancel" cc:translatable="true">Cancel</string>
 
     <!--Recovery Activity-->
-    <string name="recovery_mode_title">CommCare has entered Recovery Mode due to a problem with the local device</string>
-    <string name="recovery_mode_send_data_btn">Send Data</string>
-    <string name="recovery_mode_attempt_recovery_btn">Attempt Recovery</string>
-    <string name="recovering_resources_title">Recovering Missing Resources</string>
-    <string name="recovering_resources_progress">Installing missing resources...</string>
+    <string name="recovery_mode_title">CommCare Recovery Mode</string>
+    <string name="recovery_mode_detail">CommCare has entered recovery mode due to a problem with your app. We are trying to recover your app. Please wait...</string>
     <string name="recovery_resource_progress">Recovered %1$s out of %2$s resources</string>
-    <string name="recovery_error">We were not able to recover your app. Please make sure all forms are submitted and reinstall your app</string>
+    <string name="recovery_error">We were not able to recover your app. Please reinstall your app by going to App Manager</string>
     <string name="recovery_success">App recovery has been successful. Get back to work!</string>
```

Comment:
This is an old message, right? I think we should take out the cheeky "Back to work" part of it (I'm reminded of the cringe-y faux-casual messages Windows gives you when it dies losing all of your work)

Reply:
lol. No that was me trying to be cheeky and trying to prompt user to press back. I now think if recovery is successful we should instead show a toast and launch the DispatchActivity again instead of waiting for the user action ?

Reply:
I think it is useful for users to see that the app recovery worked, actually (feedback during recovery operations is a good UX best practice) so I'd default to leaving the screen rather than auto-directing back. I do think it's probably useful to tell the user to press 'back' or provide a UI element to resume normal functioning).

And I'm definitely fine with cheeky messages elsewhere, but we've gotten into trouble in the past with light-hearted stuff being shown to users around errors or problems, since the tone feels really off when/if people are frustrated.